.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -26,35 +25,23 @@ body:
     validations:
       required: true
   - type: textarea
-    id: screenshots
+    id: actual-result
     attributes:
-      label: Screenshots
-      description: Please add screenshots if applicable
+      label: Actual Results
+      description: Add your observations/output/log/screenshots if applicable
     validations:
       required: false
```

Comment:
Shouldn't this be "required"?

Reply:
yes, that should be marked as a required field.
Fixed :)